packages/openapi/src/decorators.ts

```diff
@@ -82,7 +82,7 @@ function isOpenAPIExtensionKey(key: string): key is ExtensionKey {
  */
 const defaultResponseKey = createStateSymbol("defaultResponse");
 export function $defaultResponse(context: DecoratorContext, entity: Model) {
-  http.setStatusCode(context.program, entity, ["*"]);
+  http.setStatusCode(context.program, entity, ["default"]);
```

Comment:
Do we want to have this concept of `default` in the http library?

Reply:
I can't say I fully understand how we decide where decorators should live. At some point in the past we thought this belonged in the openapi library. Is there some clear argument for why that prior decision was wrong? If so, I'm fine with moving it, but do we need to do that as part of this PR?